examples/threaded_example/threaded.rs

```diff
@@ -0,0 +1,529 @@
+//! Example usage of [`tss_ecdsa`] crate.
+//!
+//! Each [`Participant`] is represented by a worker thread. This example uses
+//! [`std::sync::mpsc`] channels to communicate [`Message`]s
+//! amongst threads.
```

Comment:
todo: Add documentation up here about the trust model of this example. Specifically, this example does not implement sender authentication, which is required for a secure deployment. This means that
1. The coordinator is trusted to route messages correctly.
2. The workers are trusted to not forge messages from other participants.